projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.html

```diff
@@ -1,11 +1,11 @@
 <section
   class="go-accordion-panel"
-  [ngClass]="containerClasses()"
+  [ngClass]="containerClasses"
```

Comment:
This is the big win here^ Angular will no longer update these class attribute on any javascript event.